eventmesh-registry-plugin/eventmesh-registry-zookeeper/src/main/java/org/apache/eventmesh/registry/zookeeper/pojo/Instance.java

```diff
@@ -0,0 +1,50 @@
+package org.apache.eventmesh.registry.zookeeper.pojo;
+
+
+import java.io.Serializable;
+import java.util.Map;
+
+public class Instance implements Serializable {
```

Comment:
It's better to rename to EventMeshInstance? 
And please add lombok to this module, then you can move the get/set method of the pojo.
```
compileOnly 'org.projectlombok:lombok:1.18.22'
annotationProcessor 'org.projectlombok:lombok:1.18.22'
```